add-on/panels/js/conversation.jsx

```diff
@@ -105,7 +105,6 @@ loop.conversation = (function(mozL10n) {
       ["GetAllStrings"],
       ["GetLocale"],
       ["GetLoopPref", "ot.guid"],
-      ["GetLoopPref", "textChat.enabled"],
```

Comment:
This isn't quite going to work as you might expect. The loop.requestMulti call below uses this to request the values from the browser. The results are returned in an ordered Array. In the `then` section, the array is iterated over and the values extracted.

So we actually need to remove the 

      var useDataChannels = results[++requestIdx];

as well.

I think for now, I'd like to keep the `useDataChannels` option to `loop.OTSdkDriver` available, so you can just set that to `true`.